src/java.desktop/windows/native/libawt/windows/awt_PrintJob.cpp

```diff
@@ -684,7 +685,7 @@ Java_sun_awt_windows_WPageDialogPeer__1show(JNIEnv *env, jobject peer)
                     jboolean err = setPrintPaperSize(env, self, devmode->dmPaperSize);
                     if (err) {
                         CLEANUP_SHOW;
-                        return JNI_FALSE;
+                        return doIt;
```

Comment:
This file should be identical to 7698746c179525115734bc0ed6ddc1f3ae891475; in particular, line 694 is missing:
```c
        doIt = JNI_TRUE;
```
and line 713 should be:
```c
    return doIt;
```

Reply:
Matched the upstream version.